eng/source-build-patches/0013-Add-file-to-include-missing-IsExternalInit-class.patch

```diff
@@ -0,0 +1,23 @@
+From 13eb98583ea890a52e90bd0b217771da6c93df5c Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Wed, 17 Feb 2021 17:27:41 +0000
+Subject: [PATCH] Add file to include missing IsExternalInit class
+
+See https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/init#metadata-encoding
+---
+ src/Compilers/CSharp/Portable/DummyType.cs | 4 ++++
+ 1 file changed, 4 insertions(+)
+ create mode 100644 src/Compilers/CSharp/Portable/DummyType.cs
+
+diff --git a/src/Compilers/CSharp/Portable/DummyType.cs b/src/Compilers/CSharp/Portable/DummyType.cs
+new file mode 100644
+index 00000000000..291309bdfd5
+--- /dev/null
++++ b/src/Compilers/CSharp/Portable/DummyType.cs
+@@ -0,0 +1,4 @@
++namespace System.Runtime.CompilerServices
```

Comment:
This should already be defined at https://github.com/dotnet/roslyn/blob/main/src/Compilers/Core/Portable/InternalUtilities/IsExternalInit.cs. If that isn't getting included in source-build, that's likely a problem in an of itself.

Reply:
I verified this is patch is not necessary and removed it.